CONTRIBUTING.md

```diff
@@ -127,99 +127,77 @@ This will serve the app at [localhost:4141](http://localhost:4141/), and watch f
-
-After setting up Python and Kedro, you will need to have a Kedro project setup. If you don't have any existing Kedro project, you can create a new one with the `spaceflights` example:
+### Data sources
 
-> **Note**: You can use any other [starters](https://github.com/kedro-org/kedro-starters), except `mini-kedro`, for the purpose of this development server, not just `spaceflights`.
+Kedro-Viz uses a unique identifier to determine the data source from one of several available sources. You can configure this by appending a query string to the URL, e.g. `http://localhost:4141/?data=random`. Alternatively, you can set it with an environment variable when starting up the dev server:
 
 ```bash
-cd $HOME # or wherever you prefer to keep your test project
-kedro new --starter=spaceflights
+DATA=random npm start
 ```
 
-You can use default values when creating the project, which will create a new project called `new-kedro-project`. Changing your directory into that project and install the project's dependencies:
+These are the supported dataset identifiers:
 
-```bash
-cd new-kedro-project
-kedro install
-```
+| Identifier | Data source |
+|------------|-------------|
+| `json` (default) | `/public/api/main` |
+| `random` | Randomly-generated data |
+| `demo` | `/src/utils/data/demo.mock.js` |
+| `spaceflights` | `/src/utils/data/spaceflights.mock.json` |
 
-After installing the project's dependencies, make sure you can run it with:
+By default in production, the app asynchronously loads JSON from the `/api/main` endpoint. You can replicate this in development by placing a JSON dataset in `/public/api/main`, using `main` as the name of the file, [without an extension](https://www.computerhope.com/issues/ch002089.htm). Note that operating systems often add hidden file extensions, so you might need to use a CLI to confirm the filename.
 
-```bash
-kedro run
-```
+Alternatively, you can synchronously load one of the mock datasets in `/src/utils/data`. The 'spaceflights' dataset is mainly used as mock data for unit testing.
 
-Now you are ready to launch a new viz's development server with a real Kedro project. First, changing your directory back to kedro-viz:
+Finally, you can use pseudo-random data, which is procedurally-generated on page load, and is often useful for local development. Random data can be seeded with a hash string, which will allow you to replicate a generated layout. You can supply a seed with a `seed` query string in the URL, e.g. `http://localhost:4141/?data=random&seed=oM4xauN4Whyse`. If you do not supply a seed, the app will generate a new pseudo-random one every time, and will output it to the browser console in case you wish to reuse it.
```

Comment:
Nice one - thanks for adding this to bridge the gap! Great one. 

```suggestion
Finally, you can use pseudo-random data, which is procedurally-generated on page load, and is often useful for local development. Random data can be seeded with a hash string, which will allow you to replicate a generated layout. You can supply a seed with a `seed` query string in the URL, e.g. `http://localhost:4141/?data=random&seed=oM4xauN4Whyse`. If you do not supply a seed query to the URL, the app will generate a new pseudo-random seed on every browser refresh, and will output it to the browser console in case you wish to reuse it.
```